qmcpy/integrand/um_bridge_wrapper.py

```diff
@@ -0,0 +1,51 @@
+from ._integrand import Integrand
+from ..util import ParameterError
+from numpy import *
+
+class UMBridgeWrapper(Integrand):
+    """
+    UM-Bridge Model Wrapper
+    References:
+
+        [1] UM-Bridge documentation. https://um-bridge-benchmarks.readthedocs.io/en/docs/index.html
+    """
+
+    def __init__(self, true_measure, model, config, parallel=False):
+        """
```

Comment:
Some models may not have any config parameters, and the assumption is that any umbridge model should use defaults if no config is passed. It's therefore fine to just set `config={}` as a default argument here

Reply:
Done.